### ISSUE TITLE
Dockerfile does not follow standard practice for RUN commands, leading to large image size and amount of layers

### ISSUE THREAD

**Issue by C0rn3j** [matatonic/openedai-speech#40]
```dockerfile
RUN apt-get update && apt-get install --no-install-recommends -y curl ffmpeg
RUN if [ "$TARGETPLATFORM" != "linux/amd64" ]; then apt-get install --no-install-recommends -y build-essential ; fi
RUN if [ "$TARGETPLATFORM" != "linux/amd64" ]; then curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y ; fi
RUN apt-get clean && rm -rf /var/lib/apt/lists/*
```

This creates 4 layers, and all of those files and lists are kept because they were removed and installed in separate layers.

Just convert it to a single RUN and use a heredoc for nice syntax on top:

```dockerfile
RUN <<EOR
apt-get update
apt-get install --no-install-recommends -y curl ffmpeg
if [ "$TARGETPLATFORM" != "linux/amd64" ]; then 
	apt-get install --no-install-recommends -y build-essential
fi
if [ "$TARGETPLATFORM" != "linux/amd64" ]; then 
	curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
fi
apt-get clean && rm -rf /var/lib/apt/lists/*
EOR
```

**Comment by matatonic** [comment]
Do you have a non linux/amd64 to test this on?

**Comment by matatonic** [comment]
Fixed in 0.17.3
